pages/common/grep.md

```diff
@@ -13,7 +13,7 @@
 
 - Search for a pattern recursively in the current directory, ignoring non-text files:
```

Comment:
We use this format to explain which argument applies each behavior in the command.
```suggestion
- Search for a pattern [R]ecursively in the current directory, showing matching line [n]umbers, [I]gnoring non-text files:
```

Reply:
Thanks!